### PR TITLE
{chem}[intel/2016a] CheMPS2 v1.7-rc2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 please review?

Just a version bump, really...

File: easybuild/easyconfigs/c/CheMPS2/CheMPS2-1.7-rc2-intel-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+source_urls = ['https://github.com/SebWouters/CheMPS2/archive/']
+
+builddependencies = [('CMake', '3.5.2')]
+
+dependencies = [
+    ('GSL', '2.1'),
+    ('HDF5', '1.8.16')
+]
+
+buildopts = 'VERBOSE=1'
```
Comment (wpoely86) [inline easybuild/easyconfigs/c/CheMPS2/CheMPS2-1.7-rc2-intel-2016a.eb]: not needed

**Comment by wpoely86** [comment]
:+1:

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
node2069.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/9216cd470c9b3e531f0b0896e357ab2e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2538.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/5343e467f7f716d7134556037b0ec33e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node1086.muk.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/0401348e36a5081026ef3ae7db2b509c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/6c4944f03814d866b657e59f81c4d573 for a full test report.

**Comment by boegel** [comment]
not sure why the tests were failing in the older test report, can't seem to reproduce that anymore (and nothing changed)

**Comment by boegel** [comment]
Going in, thanks for the review @wpoely86!

**Pull request merged**